se.emijoh.mpw.yml

```diff
@@ -1,40 +1,19 @@
 app-id: se.emijoh.mpw
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: mpwcli.py
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: mpw
 modules:
-  - name: mpwcli
+  - name: mpw
     buildsystem: simple
     build-commands:
-      - install -D mpwcli.py /app/bin/mpwcli.py
+      - cd go && /usr/lib/sdk/golang/bin/go build -v .
```

Comment:
will this work offline?

Reply:
It should. I use the vendoring patterns so I have all the dependencies in the repo under a vendor directory. https://github.com/emiljoha/mpw/tree/master/go/vendor

Might of course messed it up somehow :shrug: